libraries/netapp_e_series_api.rb

```diff
@@ -545,6 +544,7 @@ def status(response, expected_status_code, allowed_status_codes, failure_message
 
       # Make a call to the web proxy
       def request(method, path, body = nil)
+        require 'excon'
```

Comment:
I'd wrap this to customize the exception in the case where the user doesn't include the default recipe:

```ruby
begin
  require 'excon'
rescue LoadError
  raise "The 'excon' gem is not available. Ensure the netapp_e::default recipe is at the beginning of your run_list before attempting to use netapp_e resources."
end
```